modules/living_spaces_circles/src/Form/LivingSpacesCirclesAddPeopleForm.php

```diff
@@ -193,6 +194,23 @@ public function ajaxSubmitForm(array &$form, FormStateInterface $form_state) {
       $form['circle']['value_field']['#value'] = '';
 
       $response->addCommand(new ReplaceCommand('*[id^=living-spaces-circles-add-people-form]', $form));
+
+      // Update Members Tab.
+      $group_ids = [$group->id()];
+      if ($group->hasField('circles')) {
```

Comment:
No need to add validation. The field is added in the module for all group bundles.